src/components/PanelList.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+// Components
+import FilterInput from './FilterInput';
+import List from './List';
+
+function PanelList(props) {
+  return (
+    <div className="dashboard__panel-container">
+      <div className="dashboard__panel">
+        <div className="panel__header">
+          <h2>{props.title}</h2>
+
+          <FilterInput
+            filtering={props.filtering}
+            inputSet={props.inputSet}

```

Comment:
It's not immediately apparent that `inputSet` is an event handler. Usually our convention is to prefix handlers with `on`, followed by something descriptive, like `onFilterChanged`.

The same applies to `filterReset`.